src/sentry/utils/env.py

```diff
@@ -41,6 +41,60 @@ def log_gcp_credentials_details(logger) -> None:
     if in_test_environment():
         return
 
+    # Checking GOOGLE_APPLICATION_CREDENTIALS environment variable
+    adc_path = os.environ.get("GOOGLE_APPLICATION_CREDENTIALS", "")
+    if adc_path:
+        with open(adc_path) as fp:
+            adc = json.load(fp)
+
+            logger.info(
+                "gcp.credentials.file_found",
+                extra={
+                    "adc": adc,
```

Comment:
I think this will leak credentials into the logs? We should avoid doing that if possible, and probably use the old method of only picking out non-sensitive info?

Reply:
ok done